src/pages/Environments.js

```diff
@@ -647,8 +814,29 @@ const Environments = () => {
           loadData()
         }
       }),
-      loading && spinnerOverlay
-    ])
+      migrateDisk && h(MigratePersistentDiskModal, {
+        disk: migrateDisk,
+        workspaces: _.flow(
+          _.get(migrateDisk.googleProject),
+          _.values,
+          _.filter(({ accessLevel, workspace: { googleProject } }) => {
+            return Utils.canWrite(accessLevel) && _.isEmpty(disksByProject[googleProject])
+          }),
+          _.map('workspace'),
+          _.sortBy(({ name }) => _.lowerCase(name))
+        )(workspaces),
+        onDismiss: () => setMigrateDisk(undefined),
+        onSuccess: () => {
+          setMigrateDisk(undefined)
+          loadData()
+        },
+        contactSupport: () => contactUsActive.set(true),
+        deleteDiskId: setDeleteDiskId
+      })
+    ]),
+    h(MigratePersistentDisksBanner, { count: _.countBy('requiresMigration', disks).true }, []),
```

Comment:
Instead of having `MigratePersistentDisksBanner` render nothing if `count` is 0, it might be clearer to conditionally render this component.

```suggestion
    _.countBy('requiresMigration', disks).true > 0 && h(MigratePersistentDisksBanner, { count: _.countBy('requiresMigration', disks).true }, []),
```